src/axom/sidre/tests/sidre_native_layout.cpp

```diff
@@ -520,6 +521,146 @@ TEST(sidre_native_layout,import_conduit_and_verify_protocol)
   }
 }
 
+//------------------------------------------------------------------------------
+
+TEST(sidre_native_layout, basic_demo_compare)
+{
+  //
+  // create demo a datastore that includes:
+  //  groups
+  //  views:
+  //    scalar, string, array, external array, array of sliced buffer
+
+  DataStore ds;
+  Group* root = ds.getRoot();
+
+  // Note:
+  // we use int64 and float64 b/c those types persist even with
+  // json or yaml output
+
+  conduit::int64     sidre_vals_1[5] = {0,1,2,3,4};
```

Comment:
Minor: Do we want to use the `conduit` types, or their corresponding `axom` or `sidre` types?

Reply:
I think we have axom types that alias to the conduit types. If so, we should probably use the axom type names.

Reply:
Sure thing, which one should i use sidre or axom types?

Reply:
I'd think we should prioritize the axom types, but whatever makes it more consistent with the rest of the file would probably be best.